data_structures/linked-list/test_ll_merge.py

```diff
@@ -1,17 +1,16 @@
 from ll_merge import merge_lists
 
 
-
 def test_merge_list_equal(second_small_ll, small_ll):
-    assert merge_lists(second_small_ll, small_ll) == 4
-    assert small_ll._size == 8
+    merge_lists(second_small_ll, small_ll)
+    assert small_ll.__len__() == 8
```

Comment:
You do not need to call `__len__` explicitly. You can just use the `len` function, which calls the `__len__` method.